docs/en/command/cqueue.md

```
@@ -106,6 +107,48 @@ cqueue --format "ID:%8j | Name:%.15n | State:%t"
+| %e | ElapsedTime | Elapsed time since step started |
+| %L | NodeList | List of nodes the step is running on |
+| %t | State | Current state of the step |
+| %l | TimeLimit | Time limit for the step |
+| %N | NodeNum | Number of nodes allocated to the step |
+| %a | Account | Account (inherited from parent job) |
+| %q | QoS | Quality of Service (inherited from parent job) |
+| %o | Command | Command line of the step |
+
+**Step Format Examples:**
+
+```bash
```

markdownlint-cli2 (0.18.1)

Code fence style
Expected: tilde; Actual: backtick

(MD048, code-fence-style)